CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
+#### Enhancements
+* Rewrite the library.
+[Pranjal Satija](https://github.com/pranjalsatija)
+[#11](https://github.com/BottleRocketStudios/iOS-Holocron/pull/11)
+
 ## Master
 #### Enhancements
```

Comment:
Could you remove the duplicate `## Master` and `#### Enhancements` sections and group all the bullets together?